### ISSUE TITLE
test result not available yet for GUID QR code on a positive PCR test

### ISSUE THREAD

**Issue by hendriktews** [corona-warn-app/cwa-app-android#5318]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [ ] Bug is specific for Android only, -- cannot check this, I only have one mobile
* [X] Bug is not already reported in another issue

## Technical details

- App version: 2.23.2

## Describe the bug

I have a positive PCR test result that contains a "GUID für Corona-Warn-App" with a QR code. However, when I scan the QR code with the app, the app says that the result is not available yet, which is obviously nonsense. The test result is from yesterday 16:35 and even today the app insisted that the result were not available yet.
What I find suspicious is that the QR code scans as "https://localhost/?GUID..."

**Comment by MikeMcC399** [comment]
@hendriktews 

It sounds like the lab dealing with your test has transmitted the test result to you but it hasn't transmitted the result to the Corona-Warn-App server in parallel. The way this works is described in [CORONA-WARN-APP SOLUTION ARCHITECTURE - PCR tests: Retrieval of Lab Results and Verification Process](https://github.com/corona-warn-app/cwa-documentation/blob/main/solution_architecture.md#pcr-tests-retrieval-of-lab-results-and-verification-process).

You could try contacting the lab which sent you the positive PCR test result to ask them to resolve why you haven't received the result also in the app. There are also some suggestions in the [FAQ - Troubleshooting - General](https://www.coronawarn.app/en/faq/results/#support_general) section in addition to the generic issue open in https://github.com/corona-warn-app/cwa-documentation/issues/400.

The QR code format `https://localhost/?GUID` is correct. If you tried to scan an incorrect QR code into the app you would get an error message. The format is defined in https://update.kbv.de/ita-update/Blankoformulare/10C_OEGD/KBV_ITA_VGEX_Implementierungshinweise_10C_OEGD.pdf

![image](https://user-images.githubusercontent.com/66998419/176996438-0cbcd2a4-7d9e-45ae-bb84-15efdd955dee.png)

Best wishes for a speedy recovery!

**Comment by dsarkar** [comment]
Hi @hendriktews, thanks for reporting. As @MikeMcC399 already explained, wir your information given, the most probable cause is that the lab did no forward the result to the CWA environment. If you provide us with the details of the lab (name, address, telephone number, email), we will forward your case. Thanks. Best wishes, DS

---
Corona-Warn-App Open Source Team

**Comment by abuntu** [comment]
Hallo zusammen,

ich habe das exakt selbe Problem wie hendriktews. Das Labor bei mir: Una Diagnostik & Pharma GmbH, www.una-pharma.de. Für den Fall, dass ihr dort nachfragen wollt.

Grüße, Andreas
